src/app/services/visit.service.ts

```diff
@@ -21,4 +22,8 @@ export class VisitService {
     getVisits(patientId: string): Observable<Array<Visit>> {
         return this.http.get<Array<Visit>>(this.apiURL + '/visit/' + patientId);
     }
+
+    getBetterQualityOfImageFromStore(code: string) {
```

Comment:
the function name is too long, it will be enough just `getBetterQualityImage`

Reply:
do you mean this method in visit service?

Reply:
yes